pages/pil/dashboard/index.js

```diff
@@ -45,6 +46,8 @@ module.exports = settings => {
 
   app.use('/procedures', require('../procedures')());
 
+  app.use('/exemptions/modules', require('../modules_exempt')());
```

Comment:
this should fail lint I think, as we don't allow underscores in filenames, go for a hyphen